### PR TITLE
feat/sms-in-europe

### PR DESCRIPTION
**Pull request by JeremyDec**
ref: MANAGER-5201
Signed-off-by: Jérémy De-Cesare <jeremy.de-cesare@corp.ovh.com>
Co-authored-by: jjuret <julien.juret@gmail.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | develop
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-5201
| License          | BSD 3-Clause

ℹ️ **This is the final PR to be merged once the QA is done**.

Check the list below to see details of previous reviews:
- https://github.com/ovh/manager/pull/3577
- https://github.com/ovh/manager/pull/3629
- https://github.com/ovh/manager/pull/3643
- https://github.com/ovh/manager/pull/3695
- https://github.com/ovh/manager/pull/3702
- https://github.com/ovh/manager/pull/3677
- https://github.com/ovh/manager/pull/3692
- https://github.com/ovh/manager/pull/3693
- https://github.com/ovh/manager/pull/3694

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report test#15981.0 ■   
* Stage 1
  * lint ■   
  * SemVer ■

**Comment by ovh-cds** [comment]
CDS Report build#15981.0 ■   
* 
  * Build ■

**Comment by ovh-cds** [comment]
CDS Report build#15981.0 ■   
* 
  * Build ■

**Comment by ovh-cds** [comment]
CDS Report installation#16032.0 ✘  
* Stage 1
  * Install ✘

File: packages/manager/modules/sms/src/sms/batches/details/delete/telecom-sms-batches-details-delete.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.message-overflow {
```
Comment (Heiiji) [inline packages/manager/modules/sms/src/sms/batches/details/delete/telecom-sms-batches-details-delete.scss:1]: `.text-truncate` class on bootstrap can do the same job no ?

**Review by Heiiji — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build#16092.0 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build#16092.1 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build#16109.0 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build#16205.0 ✘  
* 
  * Build ✘

**Review by Heiiji — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report translator-push#11790.0 ✘  
* 
  * push ✘

**Review by Heiiji — APPROVED** [review]

**Pull request merged**